packages/plugin-workspace-tools/sources/commands/foreach.ts

```diff
@@ -62,6 +62,9 @@ export default class WorkspacesForeachCommand extends BaseCommand {
   @Command.Array(`--exclude`)
   exclude: Array<string> = [];
 
+  @Command.Boolean(`--public`)
```

Comment:
`--exclude-private` might be more descriptive. `public` is likely too generic.

Reply:
We could declare it `--private` with a default of `true` - [per Clipanion's rules](https://github.com/arcanis/clipanion/blob/master/sources/core.ts#L392) users would have to type `--no-private` to set it to `true`, which is the expected behavior.

Reply:
That makes more sense (especially since the initial implementation had the inverse result since I mixed up the condition). Since the `private` field is false by default it would be consistent with having to specify `no-private`.

Reply:
Done